ocs_ci/ocs/node.py

```diff
@@ -1464,3 +1456,65 @@ def wait_for_new_osd_node(old_osd_node_names, timeout=180):
     except TimeoutExpiredError:
         log.warning(f"New osd node didn't appear after {timeout} seconds")
         return None
+
+
+def add_new_node_and_label_upi_lso(
+    node_type,
```

Comment:
Here the node_type you are referring to is RHEL or RHOCS. Change the arg name reflecting it.

Reply:
Yes, this is what Elad suggested here https://github.com/red-hat-storage/ocs-ci/pull/3870#discussion_r578387705